tests/conftest.py

```diff
@@ -2518,6 +2522,8 @@ def spark() -> "SparkSession":
         .config("spark.sql.shuffle.partitions", "1")
         .config("spark.default.parallelism", "1")
         .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions")
+        .config("spark.hadoop.fs.s3a.endpoint", "http://localhost:9000")
+        .config("spark.hadoop.fs.s3a.path.style.access", "true")
```

Comment:
nit: move to where the catalog is defined
```suggestion
```

Reply:
This is actually not catalog related, but applies for every non-Iceberg table 👍